core-services/prow/02_config/_boskos.yaml

```diff
@@ -896,6 +896,10 @@ resources:
   min-count: 40
   state: free
   type: equinix-ocp-metal-quota-slice
+- max-count: 3
+  min-count: 3
+  state: free
+  type: fleet-manager-qe-quota-slice
```

Comment:
do you manually edit it? this file should be  generated automatically by running `python generate-boskos.py`